src/App.js

```diff
@@ -1,28 +1,55 @@
 import React, { useState } from 'react';
-import './App.css';
 import ReactPolling from 'react-polling'; 
+import DailyForecastCard from './Components/DailyForecastCard';
+import ForecastTitle from './Components/ForecastTitle';
+import HeatIndicator from './Components/HeatIndicator';
+import WeatherCondition from './Components/WeatherCondition';
+import WeatherHumidity from './Components/WeatherHumidity';
+import WeatherTemp from './Components/WeatherTemp';
+import WeatherDate from "./Components/WeatherDate";
+import ForecastList from './Components/ForecastList';
+import AppHeader from "./Components/AppHeader";
+import AppFooter from './Components/AppFooter';
+import WeatherIcon from "./Components/WeatherIcon";
 
 function App() {
   const apiKey = process.env.REACT_APP_WEATHER_API_KEY
   const lat =  "-23.5489";
   const lon =  "-46.6388";
   const part = "hourly,minutely";
+  const url = `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&exclude=${part}&appid=${apiKey}&units=metric`
```

Comment:
The API's base URL could be an env variable too, as the API Key.

Reply:
Yes, until the onecall part, just remove the interrogation mark and the query string paramd